File: msgraph-mail/php/src/Models/Microsoft/Graph/Sensitivity.php
<?php

namespace Microsoft\Graph\Models\Microsoft\Graph;

use Microsoft\Kiota\Abstractions\Enum;

class Sensitivity extends Enum {
    public const NORMAL = 'normal';
    public const PERSONAL = 'personal';
    public const PRIVATE = 'private';
    public const CONFIDENTIAL = 'confidential';
}

File: msgraph-mail/java/utilities/src/main/java/graphjavav4/utilities/models/microsoft/graph/Sensitivity.java
package graphjavav4.utilities.models.microsoft.graph;

import com.microsoft.kiota.serialization.ValuedEnum;
import java.util.Objects;

public enum Sensitivity implements ValuedEnum {
    Normal("normal"),
    Personal("personal"),
    Private_escaped("private_escaped"),
    Confidential("confidential");
    public final String value;
    Sensitivity(final String value) {
        this.value = value;
    }
    @javax.annotation.Nonnull
    public String getValue() { return this.value; }
    @javax.annotation.Nullable
    public static Sensitivity forValue(@javax.annotation.Nonnull final String searchValue) {
        Objects.requireNonNull(searchValue);
        switch(searchValue) {
            case "normal": return Normal;
            case "personal": return Personal;
            case "private_escaped": return Private_escaped;
            case "confidential": return Confidential;
            default: return null;
        }
    }
}
